src/registry.rs

```diff
@@ -207,12 +207,12 @@ impl Default for Registry {
 }
 
 impl Registry {
-    /// `new` creates a Registry.
+    /// Create a Registry.
     pub fn new() -> Registry {
         Registry::default()
     }
 
-    /// `register` registers a new [`Collector`](::core::Collector) to be included in metrics
+    /// Register a new [`Collector`](::core::Collector) to be included in metrics
```

Comment:
If you make a line, then some more content, rustdoc will show the first line as a summary on doc pages. Example: https://docs.rs/raft/0.3.1/raft/#structs

```
Register a new `Collect` to be included in metric collection.

It returns...
```